### ISSUE TITLE
tsc-alias incorrectly replace a module when a file with same name exists in the current directory

### ISSUE THREAD

**Issue by SwingCosmic** [justkey007/tsc-alias#199]
I have a file named  `axios.ts` . If any file includes `import axios from "axios"` in the same directory, it will be replaced to `const axios_1 = require("./axios")`, not use the dependency in `node_modules` !

`tsconfig.json` sets `baseUrl` to `"."`.

Debug log:
```
tsc-alias debug: default replacer - requiredModule:  'axios'
tsc-alias debug: default replacer - alias:
tsc-alias debug: base-url replacer - requiredModule:  'axios'
tsc-alias debug: base-url replacer - relativePath:  './'
tsc-alias debug: base-url replacer - newImportScript:  'import(".//axios")'
tsc-alias debug: base-url replacer - modulePath:  './/axios'
tsc-alias debug: replaced file with changes: 'E:/dev/work/node/test/server/axios.d.ts'
```

**Comment by olalonde** [comment]
Dupe: https://github.com/justkey007/tsc-alias/issues/197

**Comment by dvqc** [comment]
This should be closed after #218 PR
